packages/devtools/lib/src/tables.dart

```diff
@@ -383,17 +396,21 @@ class Table<T> extends Object with SetStateMixin {
       // click handler attached when we created the row instead of rebinding
       // it when rows are reused as we scroll.
       _dataForRow[tableRow.element] = dataObject;
-      void selectRow(Element row, int index) {
+      void selectRow(CoreElement coreElement, Element row, int index) {
```

Comment:
if we don't use the `coreElement` param, why do we need it?

Reply:
No - its gone.